### PR TITLE
Display or hide answer probability

### PR DESCRIPTION
**Pull request by MichaelBitard**
Related to https://github.com/etalab-ia/piaf_agent/issues/8

Result when the probabilities are hidden. I chose to kept the probability when it is too low: `Indice de confiance trop faible : 19 %. Nous ne pouvons afficher la réponse.` It does not make sense to just display `Nous ne pouvons afficher la réponse`.

I can hide the whole lines `Indice de confiance trop faible : 19 %. Nous ne pouvons afficher la réponse.` when we want to hide the probabilities

2021-05-21_09-57-51_grim

### CODE REVIEW

File: src/components/Answer.vue [hunk 1]
```diff
@@ -25,6 +25,9 @@ export default Vue.extend({
   props: [
     'answer',
   ],
+  data: () => ({
+    displayProbability: (process.env.VUE_APP_DISPLAY_PROBABILITIES ?? 'true') === 'true'
```
Comment (guillim) [inline src/components/Answer.vue:29]: not very used to this js syntax: `(A ?? B)` stands for something like `(A === B)` ?

Comment (MichaelBitard) [reply src/components/Answer.vue:29]: That's a quite new syntax indeed.

It means if this is null or undefined, then chose the other.

The nullish coalescing operator (what a barbarian name :)): https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Comment (guillim) [reply src/components/Answer.vue:29]: ok, so `??` is more `||` but without the problematic `''` or `0` values, right ?

**Comment by MichaelBitard** [comment]
yes exactly!

**Pull request merged**